ods_ci/tests/Resources/CLI/ModelServing/llm.resource

```diff
@@ -102,6 +107,11 @@ Create Secret For S3-Like Buckets
     Should Be Equal As Integers    ${rc}    ${0}
     Run Keyword And Ignore Error    Run    oc create -f ${LLM_RESOURCES_DIRPATH}/bucket_sa_filled.yaml -n ${namespace}
     Add Secret To Service Account    sa_name=${sa_name}    secret_name=${name}    namespace=${namespace}
+    IF    ${verify_ssl} == ${FALSE}
+        ${rc}    ${out}=    Run And Return Rc And Output
+        ...    oc annotate secret ${name} -n ${namespace} serving.kserve.io/s3-verifyssl='false' --overwrite
+        Should Be Equal As Integers    ${rc}    ${0}
+    END
```

Comment:
Do not use this method with ssl support in kserve we need to use below kcs.Also we will have centrlised solution of ssl certs in 2.8 so i think it is good to add this support
https://access.redhat.com/solutions/7053013

Reply:
this is a temporary workaround. Will add support to SSL